api-microservice/package.json

```diff
@@ -1,15 +1,26 @@
 {
-  "name": "api-microservice",
+  "name": "appeal-planning-decision-api",
   "version": "1.0.0",
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "start": "ts-node src/server.ts",
+    "build": "tsc --project ./"
   },
   "keywords": [],
   "author": "",
   "license": "ISC",
```

Comment:
Will probably want to add `"private": true` too

Reply:
done! :-)